From 433482a12afac571a0ddd73a9b94e6704f5540cf Mon Sep 17 00:00:00 2001
From: jinghua <jinghua@marvell.com>
Date: Mon, 22 Feb 2016 22:29:49 +0800
Subject: [PATCH 333/538] gpio: a3700: add gpio node to a3700 dtsi file

add north bridge and south bridge gpio nodes to a3700 dtsi file.

Change-Id: Ia151a29847d4f7c2ee54cf70ecd718ad22066232
Signed-off-by: jinghua <jinghua@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27839
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 42210bd..4e85714 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -45,6 +45,7 @@
  */
 
 #include <dt-bindings/interrupt-controller/arm-gic.h>
+#include <dt-bindings/gpio/gpio.h>
 
 / {
 	model = "Marvell Armada 37xx SoC";
@@ -57,6 +58,8 @@
 		ethernet0 = &eth0;
 		ethernet1 = &eth1;
 		serial0 = &uart0;
+		gpio0 = &gpio_nb;
+		gpio1 = &gpio_sb;
 	};
 
 	cpus {
@@ -205,6 +208,21 @@
 				clocks = <&sgateclk 4>;
 				status = "disabled";
 			};
+			gpio_nb: gpio@13800 {
+				compatible = "marvell,armada3700-gpio";
+				reg = <0x13800 0x30>;
+				ngpios = <36>;
+				gpio-controller;
+				#gpio-cells = <2>;
+			};
+
+			gpio_sb: gpio@18800 {
+				compatible = "marvell,armada3700-gpio";
+				reg = <0x18800 0x30>;
+				ngpios = <30>;
+				gpio-controller;
+				#gpio-cells = <2>;
+			};
 
 			usb3@58000 {
 				compatible = "generic-xhci";
-- 
1.9.1

